src/haxeLanguageServer/features/OrganizeImportsFeature.hx

```diff
@@ -0,0 +1,210 @@
+package haxeLanguageServer.features;
+
+import haxe.ds.ArraySort;
+import tokentree.TokenTree;
+import tokentree.TokenTreeBuilder;
+import haxeLanguageServer.helper.FormatterHelper;
+import haxeLanguageServer.helper.WorkspaceEditHelper;
+
+class OrganizeImportsFeature {
+	static final stdLibPackages:Array<String> = [
```

Comment:
I'm not a huge fan of hardcoding this... Can't we just sort alphabetically? I think that's what HaxeDevelop does too.

Reply:
agreed, it's not pretty, however it's the system that I use in my projects, just alphabetically wouldn't make sense for me

I guess it should be a configuration setting what kind of sort order you prefer.